test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -17,6 +17,9 @@ namespace Microsoft.ML.Scenarios
 {
     public partial class ScenariosTests
     {
+        public const string SentimentDataPath = "wikipedia-detox-250-line-data.tsv";
+        public const string SentimentTestPath = "wikipedia-detox-250-line-test.tsv";
+
```

Comment:
perhaps this should go to some relevant file ?

Reply:
you can use TestDatasets.Sentiment.trainFile, and  TestDatasets.Sentiment.testFile

---
In reply to: [245177801](https://github.com/dotnet/machinelearning/pull/2015#discussion_r245177801) [](ancestors = 245177801)